Cargo.toml

```diff
@@ -37,7 +37,7 @@ env_logger = "0.11.5"
 proptest = "1.5.0"
 ron = "=0.9.0-alpha.0"
 varisat = "0.2.2"
-version-ranges = { version = "0.1.0", path = "version-ranges", features = ["proptest"] }
```

Comment:
Why does this version number need to be bumped?  PubGrub  does not rely on any of the new functionality.

Reply:
Normally i just bump out of convenience patch versions out of convenience (we don't need to test something older if the bound is on latest).

We should consider setting up CI that checks that pubgrub works with the lowest crates.io version-ranges, if we aren't on a version that isn't released to crates.io yet.